src/components/app-sidebar.tsx

```diff
@@ -28,14 +28,35 @@ const userOptions = [
 ];
 
 const quotationOptions = [
+  {
+    title: "Recibir",
+    url: "/proposals/list",
+    icon: Inbox,
+  },
   {
     title: "Cotizar",
-    url: "/dashboard/user-registration",
+    url: "/proposals/registration",
     icon: Search,
   },
   {
     title: "Enviar",
-    url: "/dashboard/user-list",
+    url: "/proposals/list",
+    icon: ArrowRight,
+  },
+];
+
+const reportesOptions = [
```

Comment:
Estamos usando ingles para nombrar a las variables, en este caso tenemos una mezcla de ambos, quizás se podría mejorar de la siguiente forma:

```suggestion
const reportsOptions = [
```